package.json

```diff
@@ -44,7 +44,7 @@
     "lerna": "lerna",
     "lint": "lerna run lint --no-sort --stream -- -- -- --color",
     "lint:fix": "lerna run lint:fix --no-sort --stream -- -- -- --color",
-    "policy-check": "flub check policy --exclusions build-tools/packages/build-tools/data/exclusions.json -D assert-short-codes",
+    "policy-check": "flub check policy --exclusions build-tools/packages/build-tools/data/exclusions.json",
     "policy-check:asserts": "flub check policy --exclusions build-tools/packages/build-tools/data/exclusions.json -d assert-short-codes --fix",
```

Comment:
is this line wrong as well? i don't know what the command line syntax for this is? is it documented somewhere?

Reply:
Docs are at https://github.com/microsoft/FluidFramework/tree/main/build-tools/packages/build-cli#fluid-toolsbuild-cli. `-d assert-short-codes` seems correct, it's the opposite of `-D`.

Reply:
I replaced the shortcut flags with their full forms so it's a little more self-documenting.